tests/test_routes_canvas.py

```
@@ -2,32 +2,64 @@
-    })
-    assert resp.status_code == 200
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+
+def _make_client(tmp_path):
+    from tinyagentos.app import create_app
+
+    config = {
+        "server": {"host": "0.0.0.0", "port": 6969},
```

Ruff (0.16.0)

[error] Possible binding to all interfaces

(S104)